### PR TITLE
[Enhancements] Responsive automation

### PR DESCRIPTION
**Pull request by wadhawh**
This PR includes the following changes;

- Responsive automation cases
- Automation cases for old features
- CI/CD fails fix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cypress/e2e/unauthUserCanAccessSimulation.cy.js [hunk 1]
```diff
@@ -2,36 +2,23 @@
...
 describe("Unauth Simulation", () => {
-	beforeEach(() => {
-		cy.visitDomain(`${Cypress.env("WEB_DOMAIN")}/demo`);
+	context("Desktop view", () => {
+		beforeEach(() => {
+			cy.visitDomain(`${Cypress.env("WEB_DOMAIN")}/demo`);
+		});
+	
+		it("US-001 - should allow user to use unauth simulation", () => {
+			cy.useUnauthSimulation(false)
```
Comment (mbalfour-amzn) [inline cypress/e2e/unauthUserCanAccessSimulation.cy.js:11]: Can this be changed to take in an enum? This currently reads like "don't use unauth simulation", not "use unauth simulation, but not with responsive view". An enum would make this more readable.
Ex: `cy.useUnauthSimulation(DesktopView)`

This applies to all of the functions taking in a boolean right now to switch between DesktopView and ResponsiveView.

File: cypress/e2e/userIsAbleToUseGeofence.cy.js [hunk 2]
```diff
@@ -4,86 +4,29 @@
...
-		cy.wait(30000);
-		cy.get('[data-testid="hamburger-menu"]').click();
-		cy.wait(5000);
-		cy.get('[data-testid="disconnect-aws-account-button"]').click();
+	context("Responsive view", () => {
+		const geofenceName = faker.random.word();
+
+		beforeEach(() => {
+			cy.visitDomainInResponsiveView(`${Cypress.env("WEB_DOMAIN")}/demo`);
+			cy.connectAwsAccount(true,);
```
Comment (mbalfour-amzn) [inline cypress/e2e/userIsAbleToUseGeofence.cy.js:25]:
```suggestion
			cy.connectAwsAccount(true);
```

File: cypress/e2e/unauthUserCanAccessSimulation.cy.js (hunk 1 above)
Comment (its-aazizi) [reply cypress/e2e/unauthUserCanAccessSimulation.cy.js]: resolved

File: cypress/e2e/userIsAbleToUseGeofence.cy.js (hunk 2 above)
Comment (its-aazizi) [reply cypress/e2e/userIsAbleToUseGeofence.cy.js]: resolved

**Comment by its-aazizi** [comment]
@mbalfour-amzn comments resolved

**Review by mbalfour-amzn — APPROVED** [review]

**Pull request merged**